tests/jax/components/executing/action_selection.py

```diff
@@ -0,0 +1,237 @@
+
+    Returns:
+        ExecutorSelectActionConfig
+    """
+    config = ExecutorSelectActionConfig()
+    config.parm_0 = 1
+    return config
+
+
+@pytest.fixture
+def dummy_empty_config() -> ExecutorSelectActionConfig:
+    """Dummy empty config attribute for FeedforwardExecutorSelectAction class
+
+    Returns:
+        ExecutorSelectActionConfig
+    """
+    return ExecutorSelectActionConfig()
+
+
+@pytest.fixture
+def mock_empty_executor() -> Executor:
+    """Mock executore component with empty observations"""
+    store = SimpleNamespace(is_evaluator=None, observations={})
+    return Executor(store=store)
+
+
+def get_action(parm_0, parm_1, parm_2):
+    """Function used in the networks.
+
+    Returns:
+        action_info and policy info
+    """
+    return "action_info_after_get_action", "policy_info_after_get_action"
+
+
+class MockExecutor(Executor):
+    def __init__(self):
+        observations = {
+            "agent_0": "observation_0",
+            "agent_1": "observation_1",
+            "agent_2": "observation_2",
+        }
+        agent_net_keys = {
+            "agent_0": "network_agent",
+            "agent_1": "network_agent",
+            "agent_2": "network_agent",
+        }
+        networks = {
+            "networks": {
+                agent_net_keys["agent_0"]: SimpleNamespace(get_action=get_action),
+                agent_net_keys["agent_1"]: SimpleNamespace(get_action=get_action),
+                agent_net_keys["agent_2"]: SimpleNamespace(get_action=get_action),
+            }
+        }
+        key = jax.random.PRNGKey(5)
+        action_info = "action_info_test"
+        policy_info = "policy_info_test"
+
+        store = SimpleNamespace(
+            is_evaluator=None,
+            observations=observations,
+            observation=SimpleNamespace(observation=[0, 0, 1, 0], legal_actions=3),
+            agent="agent_0",
+            networks=networks,
+            agent_net_keys=agent_net_keys,
+            key=key,
+            action_info=action_info,
+            policy_info=policy_info,
+        )
+        self.store = store
+
+    def select_action(
+        self, agent: str, observation: NestedArray, state: NestedArray = None
+    ) -> NestedArray:
+        action_info = "action_info_" + str(agent) + "_" + str(observation)
+        policy_info = "policy_info_" + str(agent) + "_" + str(observation)
+        return action_info, policy_info
+
+
+@pytest.fixture
+def mock_executor() -> Executor:
+    """Mock executore component."""
+    return MockExecutor()
+
+
+# Test initiator
+def test_constructor(dummy_config: type) -> None:
+    """Test adding config as an attribute
+
+    Args:
+        dummy_config
+    """
+    ff_executor_select_action = FeedforwardExecutorSelectAction(config=dummy_config)
+    assert ff_executor_select_action.config.parm_0 == dummy_config.parm_0
+
+
+def test_constructor_empty_config(dummy_empty_config: type) -> None:
```

Comment:
This test is not necessary and can be removed. The above test already checks if the config is loaded properly